html/themes/custom/common_design_subtheme/templates/views/views-view--indicators.html.twig

```diff
@@ -41,9 +41,7 @@
   ]
 %}
 
-{% if display_id != 'assessments_map' %}
-  {% include '@cd-components/cd-export-button/cd-export-button.html.twig' %}
-{% endif %}
+{% include '@cd-components/cd-export-button/cd-export-button.html.twig' %}
```

Comment:
@attiks This is in place now, like how we have it on AR.
I'm not sure what else was needed to provide the functionality. Maybe a custom module?